### PR TITLE
2190-RPackagecategoryName-should-deprecated-and-just-use-name

### LINKED ISSUE #2190: RPackage>>categoryName should deprecated and just use name
Let us do another pass and clean...

RPackage >> categoryName
    self deprecated rewriting....
    ^ self name

So clients should call name directly.

### PR DESCRIPTION
**Pull request by apierr**
Fixes #2190

### CODE REVIEW

File: src/Deprecated80/RPackage.extension.st [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Extension { #name : #RPackage }
+
+{ #category : #'*Deprecated80' }
+RPackage >> categoryName [
+	self
+		deprecated: 'This method will be removed. Clients should call name directly'.
```
Comment (jecisc) [inline src/Deprecated80/RPackage.extension.st]: We could have an automatic rewrite for this one.

**Review by jecisc — CHANGES_REQUESTED** [review]

File: src/Deprecated80/RPackage.extension.st (hunk 1 above)
Comment (MarcusDenker) [reply src/Deprecated80/RPackage.extension.st]: yes, that would be good.

Comment (jecisc) [reply src/Deprecated80/RPackage.extension.st]:
```
transformWith: '`@receiver categoryName' -> '`@receiver name'
```

**Comment by MarcusDenker** [comment]
Bootstrap crashes with:

[32] MessageNotUnderstood: RPackage>>categoryName
[32] RPackage(Object)>>doesNotUnderstand: #categoryName
[32] RPackageOrganizer>>validateCanBeAddedPackage:

**Comment by MarcusDenker** [comment]
I think we need to fix al the callers that are needed during the image build process.

First one to fix is the on evalidateCanBeAddedPackage:, then running the tests again will show if that is enough

**Review by MarcusDenker — APPROVED** [review]
ok

**Pull request merged**